_data/sidebars/zh_lb2_sidebar.yml

```diff
@@ -1,5 +1,899 @@
 entries:
-- title: sidebar
-  product: LoopBack - 中文
-  home_url: /doc/zh/lb2/index.html
-  folders:
+
```

Comment:
I just updated the format of this file in #73 but the difference is not major, just a couple substitutions.